apps/accounts/utils/general.py

```diff
@@ -95,10 +100,10 @@ def set_random_password(user, password_length=16):
 
 def email_to_username_hash(email):
     """Convert emails to hashed versions where we store them in the username field
-    We can't just store them directly, or we'd be limited to Django's username <= 30 chars limit,
-    which is really too small for arbitrary emails
+    We can't just store them directly, or we'd be limited to Django's username <= 30
```

Comment:
please revert, this is breaking docstrings in the middle of the thought/phrase, and makes the docstring harder to understand.